### ISSUE TITLE
Active tab

### ISSUE THREAD

**Issue by juanpprieto** [AT-UI/at-ui#76]
Hey guys,

Thanks for the great work. I've been playing around with the library and got stuck trying to activate a given tab pane on load. Is there a way to do this via props or should I just use the active tab class?

Thanks!

**Comment by koppthe** [comment]
sorry, i'm not very clear about your question. 

You want to active custom tab via props? such like active `Tab2` below?
```html
<at-tabs>
  <at-tab-pane label="Tab1" name="name1">
    <p>Content of Tab Pane 1</p>
  </at-tab-pane>
  <at-tab-pane label="Tab2" name="name2">
    <p>Content of Tab Pane 2</p>
  </at-tab-pane>
  <at-tab-pane label="Tab3" name="name3">
    <p>Content of Tab Pane 3</p>
  </at-tab-pane>
</at-tabs>
```

**Comment by juanpprieto** [comment]
Hi there,

Yes, I would like to know how to programatically activate a given tab. 

I've seen other frameworks exposing an 'active' prop to do this. 

Any tips?

**Comment by diego-lipinski-de-castro** [comment]
Yes, it`s in the docs, you can do two-way data binding by settings the v-model attribute in the <at-tabs> with the value of the name of one of the <at-tab>`s that are inside it

**Comment by juanpprieto** [comment]
Thanks @diego-lipinski-de-castro that worked.

**Comment by diego-lipinski-de-castro** [comment]
This is not working anymore, what happened?
